com.github.wwmm.PulseEffects.json

```diff
@@ -0,0 +1,267 @@
+            "-DCMAKE_INSTALL_LIBDIR=lib",
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DBUILD_SHARED_LIBS=ON",
+            "-DBUILD_TESTING=OFF",
+            "-DCMAKE_Fortran_COMPILER=/usr/lib/sdk/gfortran-62/bin/gfortran",
+            "-DLAPACKE=ON",
+            "-DCBLAS=ON"
+          ],
+          "sources": [{
+            "type": "archive",
+            "url": "http://www.netlib.org/lapack/lapack-3.7.1.tgz",
+            "sha256": "f6c53fd9f56932f3ddb3d5e24c1c07e4cd9b3b08e7f89de9c867125eecc9a1c8"
+          }]
+        },
+        {
+          "name": "numpy",
+          "buildsystem": "simple",
+          "build-commands": [
+            "PATH=$PATH:/usr/lib/sdk/gfortran-62/bin python3 setup.py build -j 0",
+            "python3 setup.py install --prefix=/app --root=/ --optimize=1"
+          ],
+          "build-options": {
+            "env": { "ATLAS": "None", "BLAS": "/app/lib", "LAPACK": "/app/lib" }
+          },
+          "sources": [{
+            "type": "archive",
+            "url": "https://github.com/numpy/numpy/releases/download/v1.13.1/numpy-1.13.1.tar.gz",
+            "sha256": "de020ec06f1e9ce1115a50161a38bf8d4c2525379900f9cb478cc613a1e7cd93"
+          }],
+          "cleanup": ["*.h", "/bin", "libnpymath.a", "*/core/include"]
+        },
+        {
+          "name": "scipy",
+          "buildsystem": "simple",
+          "build-commands": [
+            "PATH=$PATH:/usr/lib/sdk/gfortran-62/bin python3 setup.py build",
+            "python3 setup.py install --prefix=/app --root=/ --optimize=1"
+          ],
+          "build-options": {
+            "ldflags": "-shared",
+            "env": { "ATLAS": "None", "BLAS": "/app/lib", "LAPACK": "/app/lib" }
+          },
+          "sources": [{
+            "type": "archive",
+            "url": "https://github.com/scipy/scipy/releases/download/v0.19.1/scipy-0.19.1.tar.xz",
+            "sha256": "0dca04c4860afdcb066cab4fd520fcffa8c85e9a7b5aa37a445308e899d728b3"
+          }]
+        },
+        {
+          "name": "ladspa-sdk",
+          "make-args": ["targets"],
+          "make-install-args": [
+            "INSTALL_BINARY_DIR=/app/bin",
+            "INSTALL_INCLUDE_DIR=/app/include",
+            "INSTALL_PLUGINS_DIR=/app/lib/ladspa"
+          ],
+          "no-autogen": true,
+          "subdir": "src",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "http://www.ladspa.org/download/ladspa_sdk_1.13.tgz",
+              "sha256": "b5ed3f4f253a0f6c1b7a1f4b8cf62376ca9f51d999650dd822650c43852d306b"
+            },
+            {
+              "type": "patch",
+              "path": "ladspa-sdk-fix-memleak.patch",
+              "strip-components": 0
+            },
+            {
+              "type": "shell",
+              "commands": ["sed -i 's|mkdirhier|mkdir -p|' src/makefile"]
+            }
+          ],
+          "cleanup": ["/bin"]
+        },
+        {
+          "name": "gstreamer",
```

Comment:
Unfortunately not since version 1.4.3 of PulseEffects. They develop at quite a rapid pace. This is the main reason I thought this Flatpak would be a good idea on LTS distributions.